src/Akka.Hosting/AkkaHostingExtensions.cs

```diff
@@ -77,6 +77,34 @@ public static IServiceCollection AddAkka(this IServiceCollection services, strin
 
             return services;
         }
+        
+        public static IServiceCollection AddAkka<T>(this IServiceCollection services, string actorSystemName, Action<AkkaConfigurationBuilder, IServiceProvider> builder) where T:AkkaHostedService
```

Comment:
Since the generic code is identical to the non-generic one, we should just call the generic one from inside the non-generic extension method. Saves us the time to have to check that both implementation are the same in the future.